src/demo/components/ComponentPage/ComponentPage.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from 'react';
+import { Button, ButtonType } from '../../../Button';
+import { css } from '../../../utilities/css';
+import {
+  Link
+} from '../../../index';
+import './ComponentPage.scss';
+
+export interface IComponentPageProps {
+  title: string;
+  componentName: string;
+  exampleCards: Array<JSX.Element>;
+  propertiesTables?: Array<JSX.Element>;
+  bestPractices?: JSX.Element;
+  dos?: JSX.Element;
+  donts?: JSX.Element;
+  overview: JSX.Element;
+  route: string;
+}
+
+export interface IComponentPageState {
+}
+
+export class ComponentPage extends React.Component<IComponentPageProps, IComponentPageState> {
+
+  constructor(props: IComponentPageProps) {
+    super(props);
+  }
+
+  public render() {
+
```

Comment:
You can use destructuring here:

let { title, componentName, etc } = this.props;

Then just use those in your template. It reduces some redundancy in the final output.